pkgs/applications/office/notes/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, wrapQtAppsHook
+, qtbase
+}:
+
+stdenv.mkDerivation rec {
+  pname = "notes";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "nuttyartist";
+    repo = "notes";
+    fetchSubmodules = true;
```

Comment:
In nixpkgs, we try to unbundle submodules.  Could you do that?

Reply:
How?

Reply:
Now trying to fix this.

Reply:
I'm trying to package `qautostart`, but I'm unable to compile it yet... I'm going to dedicate one hour and I'll keep you up to date of any findings.

Reply:
Ok I asked a bit of help on [Matrix](https://matrix.to/#/#macos:nixos.org) and they suggested me to not package it until the next time an app needs `qautostart`.

Also, `qautostart` is, quoting someone from that channel, an abomination to package... so I think we should keep it as it is.

Ok for you @jian-lin ?

Reply:
Agreed.